schematic/models/validate_attribute.py

```diff
@@ -1462,15 +1568,25 @@ def _run_validation_across_targets_set(
                     # If there are missing values in the target manifest, log the manifest
                     # and the missing values.
                     missing_manifest_log[target_manifest_id] = missing_values
-
-        return missing_manifest_log, present_manifest_log, repeat_manifest_log
+        target_attribute_in_manifest_list.append(target_attribute_in_manifest)
```

Comment:
This isn't part of the pull request but line 1566:
`present_manifest_log.append(target_manifest_id)` 
is a dictionary which you can't append to.